.github/workflows/BUILD_ON_DEMAND.yml

```diff
@@ -23,7 +23,7 @@ jobs:
     - name: Use Node.js
       uses: actions/setup-node@v4
       with:
-        node-version: 20
+        node-version: 22
```

Comment:
OK so this is required due to `@electron/notarize` engine dependency.

Reply:
Nevermind, 24 is not yet there 🙈

Reply:
It's coming [in 2 weeks](https://github.com/nodejs/node/issues/57057), but we don't want to live on the edge.
I think we can manage node@22 here.